.github/workflows/ci.yml

```
@@ -24,6 +24,11 @@ jobs:
         run: npm run ${{ matrix.target }}
         shell: bash
 
+      - name: Publish any commit / build for testing
+        run: npm run release:pkg-pr-new
```

This should run once for the build 
```suggestion
      - name: Publish any commit / build for testing
        if: ${{ matrix.target == 'ci:build' }}
        run: npm run release:pkg-pr-new
```

---

good catch!